setup.py

```diff
@@ -45,11 +45,13 @@
     # PyJWT has loose dependency. We want the latest one.
     "cryptography==2.9.2",
     "pip>=8.0.3",
+    "pipdeptree>=1.0.0",
```

Comment:
Should we add these libs now here, or should we add some logic to check if these are available when running the script and fail the script if not available.

Reply:
Maybe add them to `requirements_test.txt` as they are used for testing/development?